integration-tests/mysql-client-tests/first-hour-tests.bats

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bats
+
+setup() {
```

Comment:
I'd recommend naming the BATS file something like `sakilla-data-dump-load.bats` just to be explicit about what's in there. That'll probably make more sense in a year or two. I'd put these bats files into a subdirectory called something like `data-dump-loading` in the mysql-client-tests directory, too. Or... you might even consider a separate directory, since these tests seem different from the other tests in mysql-client-tests that are really testing the mysql client interfaces across multiple languages.